### ISSUE TITLE
Remove 'Fallback' in css font-family list

### ISSUE THREAD

**Issue by nancym** [AngeloStavrow/indigo#38]
First, thank you for this nice theme with no third-party dependencies!

`static/css/style.css` contains lines like this:

```
font-family: 'Charter', Fallback, serif;
```

And `layouts/partials/fonts.css` contains font-family 'Charter' and others, but no font-family named Fallback. Where is Fallback specified or does it have a special meaning in CSS that I don't know about?

Thanks for any info about the Fallback font-family.

**Comment by AngeloStavrow** [comment]
That's a good catch, @nancym — I spent more time trying to get the fonts loading than actually paying attention to how I was writing them!

I used [this article](https://css-tricks.com/snippets/css/using-font-face/) to sort that out and it looks like I just copy/pasted the font declarations instead of actually defining a fallback. I will remove `Fallback` from any `font-family` declarations in the next release.
